### PR TITLE
Adds the alternate_farts var to mobs

### PR DESCRIPTION
**Pull request by ghost**
Now admemes can varedit your farts, too!

:cl: Thefastfoodguy
admin: Adds the alternate_farts var to mobs
/:cl:

### CODE REVIEW

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -1,3 +1,6 @@
+/mob/living
+	var/list/alternate_farts
```
Comment (KayeArray) [inline hippiestation/code/modules/mob/living/carbon/human/emote.dm:2]: shouldn't you init this list?

**Comment by Arzion5** [comment]
What a time to be alive +1

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/modules/mob/living/carbon/human/emote.dm:2]: done (it's done as a LAZYINITLIST in the code)

**Comment by mlgdatboi** [comment]
+1 not exploitable in any way as only admins can add them

**Review by GuyonBroadway — DISMISSED** [review]
Yeah i guess this looks good.

**Comment by GuyonBroadway** [comment]
build this

**Review by tortellinitony — CHANGES_REQUESTED** [review]
Integrate alien fart sound with this system.

**Comment by tortellinitony** [comment]
Set alien mobs `alternate_farts = list('hippiestation/sound/effects/alienfart.ogg')`

File: hippiestation/code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -64,8 +67,12 @@
 			"farts so loud it startles them!",
 			"breaks wind and a nearby wine glass!",
 			"<b>finally achieves the perfect fart. All downhill from here.</b>")
+	LAZYINITLIST(user.alternate_farts)
+	if(LAZYLEN(user.alternate_farts))
+		fartsound = pick(user.alternate_farts)
 	if(istype(user,/mob/living/carbon/alien))
-		fartsound = 'hippiestation/sound/effects/alienfart.ogg'
+		if(!LAZYLEN(user.alternate_farts))
```
Comment (tortellinitony) [inline hippiestation/code/modules/mob/living/carbon/human/emote.dm]: Remove this bit here

**Review by tortellinitony — DISMISSED** [review]

**Review by j-mie — APPROVED** [review]

**Pull request merged**